script.js

```diff
@@ -0,0 +1,200 @@
+const resume = {
+    "basics": {
+        "name": "Julia Dufresne",
+        "label": "Web Developer",
+        "picture": "",
+        "email": "juliadufresne@gmail.com",
+        "phone": "(306) 681-4881",
+        "website": "http://juliadufresne.com",
+        "summary": "Former Content Designer & Developer who found a passion for coding on the job. Through various education streams, I am now a Web Developer looking to utilize my eye for design to build applications with the user's experience and accessibility as top priority.",
+        "location": {
+        "city": "Toronto",
+        "countryCode": "Canada",
+        "region": "Ontario"
+        },
+        "profiles": [{
+        "network": "Twitter",
+        "username": "julesdufresne",
+        "url": "https://twitter.com/julesdufresne"
+        }]
+    },
+    "work": [{
+        "company": "Continental Cosmetics Ltd",
+        "position": "Content Designer & Developer",
+        "startDate": "December 2018",
+        "endDate": "January 2020",
+        "summary": {
```

Comment:
I believe you don't need the "summaryOne" and "summaryTwo" here (or in any of the summary sections). I think you should be able to reference them with for loops like you have below in your code, but this way it would be more dynamic for other json inputs.

ex:
```
"work": [{
...
"summary": {
"Utilized HTML, CSS, Adobe Creative Cloud and MailChimp to design and develop emails, banners and flyers in a cosmetic distribution company to promote brands, products, education and events to spas and salons Canada-wide.",
"Revenue had positive growth each quarter, our education and events had a higher number of attendees, and an increasing number of spas and salons were taking on our brands."
},
}
...
]
```